Fixed cors ascii rick roll instead of opening gif in new window

jandobrx opened the pull request:
We saw that a recent [PR](https://github.com/rootvc/cli-website/pull/84) featured a rick roll curl, thought it was a great addition but had a CORS issue. Wanted to fix it!

![Old](https://github.com/rootvc/cli-website/assets/134740746/a4cc50a1-ef48-4333-ae7c-17693fb1c51f)
Updated Xterm to the new version, fixed cursor position bug after setCurrentLine(), added a video that appends in the DOM.
![New](https://github.com/rootvc/cli-website/assets/134740746/c65aff55-d1d8-4a47-9f45-57c8c582c8e9)

Test Netlify Deployment w/changes:
https://main--test-cli-rick.netlify.app/

gurgenbrx committed:
Updated to new xterm, added rickroll code

gurgenbrx committed:
Added video

gurgenbrx committed:
Readded ai dependency

gurgenbrx committed:
Added the new package lock

ledwards commented:
omg amazing

This has been on my feature list for over a year and never really gotten around to it. Thank you!!

Your test deployment shows it working on desktop for me (Chrome and Safari for Mac) but not getting it to work on mobile (mobile Safari) - it shows Nice try but then a bunch of blank lines.

Given we're a VC website, iOS is actually our #1 platform haha - this is so close, I'll try to dig into debugging it on mobile Safari and then I 100% want to merge. Thank you!!

gurgenbrx committed:
Video to AA Frame Array

This commit switches from injecting a video, to using a 54 frame ASCII Rick Roll.
Note that Mobile and Desktop versions contain two different AA Arrays due to resolution constrains.

gurgenbrx committed:
Fixed isMobile boolean logic

gurgenbrx committed:
Removed an unnecessary if statement

gurgenbrx committed:
Added the dep gen video
